adaptors/library/jobs/satusehat-create-encounter.js

```diff
@@ -0,0 +1,101 @@
+// Create an encounter in Satusehat
```

Comment:
Maybe this should be like `Create an Encounter  in Satusehat, based on an incoming Commcare visit`

We don't have to explain the Commcare visit structure or anything, but it'll really help people understand what's happening if they have some frame of reference for what the input structure represents.

I suppose that context could be set by the example name too